### ISSUE TITLE
Dark Theme turn into White Theme when re-open the GUI

### ISSUE THREAD

**Issue by cxbignekoc** [greatest-ape/OctaSine#76]
**Describe the bug**

the dark theme switched back into white theme when the interface is re-opened

**What did you try to do**
1. i load the plugin into bitwig 4.0.7
2. click on `theme` it turns in dark mode
3. close/minimize the plugin
4. open the plugin via "show plugin window" from the wrapper
5. the theme is now white

**Expected behavior**
dark theme

**Screenshots**
no

**Software (please complete the following information):**
 - OctaSine version [0.7.0].
 - DAW [Bitwig 4.0.7]
 - OS: [Windows 10 Pro]

**Log file contents**
```
11:34:58 [INFO] init
11:34:58 [INFO] OS: Windows 10.0.19043 (Windows 10 Pro) [64-bit]
11:34:58 [INFO] OctaSine build: v0.7.0 (b05fcf1) (gl)
```
cheers*

**Comment by greatest-ape** [comment]
Thanks, I’ll look into it.

**Comment by greatest-ape** [comment]
@cxbignekoc Does this build solve the issue?

[OctaSine-tmp-2022-07-31-Windows.zip](https://github.com/greatest-ape/OctaSine/files/9228963/OctaSine-tmp-2022-07-31-Windows.zip)

**Comment by greatest-ape** [comment]
There’s been an additional fix in 0.8.0, so I’m closing this now.
